### PR TITLE
feat: Run specific seed file (closes #801)

### LINKED ISSUE #801: Seed a single file!
I dont see any option for seeding a single file.
For ex I have folder called "development" where I have all seed files.
How do I seed only a single file from that folder?

knex seed:run --env=development  
this command does seeding for all files on development folder

### PR DESCRIPTION
**Pull request by felixmosh**
This features allows to run a specific seed file.
It adds an additional cli property `--specific=filename.js` which points to some seed file in the seed directory.

usage: `npx knex seed:run --specific=seed-filename.js`

### CODE REVIEW

File: test/integration/seed/index.js [hunk 1]
```diff
@@ -5,23 +5,32 @@ const path = require('path');
...
+    it('should run all seed files in the configured seed directory', () => {
       return knex.seed
         .run({ directory: 'test/integration/seed/test' })
         .then(([data]) => {
           expect(path.basename(data[0])).to.equal('seed1.js');
           expect(path.basename(data[1])).to.equal('seed2.js');
         });
     });
+
+    it('should run specific seed file in the configured seed directory', () => {
```
Comment (kibertoad) [inline test/integration/seed/index.js]: async-awaitis preferable for new tests

**Comment by kibertoad** [comment]
Could you please also create documentation pr in knex documentation repo?

**Comment by felixmosh** [comment]
> Could you please also create documentation pr in knex documentation repo?

I've added the additional "sections" in the documentation, but when run dev / build and serving the docs, I don't see the additions even though I do see the additions in the source code of the html.

Any help?

**Comment by felixmosh** [comment]
Ok, managed to fix the issue, apparently there is `mssql` deps without it to be in `package.json`.

Added: https://github.com/knex/documentation/pull/213

File: test/integration/seed/index.js (hunk 1 above)
Comment (felixmosh) [reply test/integration/seed/index.js]: done

**Comment by kibertoad** [comment]
Thank you!

**Pull request merged**